Sources/TuistGenerator/Linter/GraphLinter.swift

```diff
@@ -296,6 +342,7 @@ public class GraphLinter: GraphLinting {
             LintableTarget(platform: .iOS, product: .framework),
             LintableTarget(platform: .iOS, product: .staticFramework),
             LintableTarget(platform: .iOS, product: .bundle),
+            LintableTarget(platform: .iOS, product: .appClip),
```

Comment:
I'm a bit confused. In which scenario does a unit test target depend on an app clip target?

Reply:
if you want to add unit tests for appclip